src/foraging_gui/Foraging.py

```diff
@@ -303,7 +303,7 @@ def _StageSerialNum(self):
                 try:
                     instance.io.close()
                 except Exception as e:
-                    logging.error(str(e))
+                    pass#logging.error(str(e))
```

Comment:
What happens if we have 
> logging.error(str(e))
> pass

instead of just
> pass

Reply:
I haven't tried this, but my speculation is a similar effect.

Reply:
Yes, I tested it. The same issue

Reply:
Ok, this issue is unrelated to the overall reward delay. So we can deal with it separately.